### PR TITLE
Support acorn 6 and 7 as peerDependencies

### LINKED ISSUE #100: Acorn v7.0
Acorn have recently released a new major version: https://github.com/acornjs/acorn/releases/tag/7.0.0  
Could you please bump your dependencies?

### PR DESCRIPTION
**Pull request by goto-bus-stop**
Fixes #100 

Runs tests with both v6 and v7, so it doesn't need a major bump in acorn-jsx.

### CODE REVIEW

**Comment by bjankord** [comment]
Would be nice to get this in 👍

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,13 @@
 language: node_js
-sudo: false
```
Comment (RReverser) [inline .travis.yml]: Why was this removed?

File: .travis.yml [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
   - '6'
-  - '7'
+  - '8'
+  - '10'
+  - '12'
+env:
+  - ACORN_VERSION=6
+  - ACORN_VERSION=7
+
+before_script:
```
Comment (RReverser) [inline .travis.yml]: This would be better done as override of `install` step. Then Travis won't run `npm install`, and you can just install the required peer dependency unconditionaly.

File: .travis.yml (hunk 1 above)
Comment (goto-bus-stop) [reply .travis.yml]: AFAIU this effectively became a noop as of https://changelog.travis-ci.com/the-container-based-build-environment-is-fully-deprecated-84517

File: .travis.yml (hunk 2 above)
Comment (goto-bus-stop) [reply .travis.yml]: I did it this way because doing `npm install acorn@7` on a newly cloned repo will only install acorn, not any of the package's dependencies.

Comment (RReverser) [reply .travis.yml]: Well we don't have any other deps aside from `acorn`, so it should be fine.

Comment (goto-bus-stop) [reply .travis.yml]: hah, that's fair enough. I might've been wrong in my last comment then, because I only tested it in my acorn-jsx clone! will update

**Comment by RReverser** [comment]
Published as 5.0.2.
